### PR TITLE
Add integration tests using runtest

### PR DESCRIPTION
**Pull request by stigrj**
Added a small test suite with four different runs

### CODE REVIEW

**Comment by stigrj** [comment]
"No module named runtest"? :thinking:

File: .travis.yml [hunk 1]
```diff
@@ -46,4 +46,5 @@ script:
   - cd "$WHICH"
   - cmake -H. -Bbuild
   - cmake --build build -- VERBOSE=1
-  # Testing
+  - cd build
+  - ctest --output-on-failure --verbose
```
Comment (robertodr) [inline .travis.yml]: I think you need to run within the Pipenv shell: `pipenv run ctest ...`

**Comment by stigrj** [comment]
:confetti_ball:

**Review by robertodr — APPROVED** [review]

**Comment by bast** [comment]
Did I hear runtest? Cool!

**Comment by stigrj** [comment]
Pseudo-random numbers are of course different on mac

**Comment by robertodr** [comment]
Is the starting seed the same on all platforms?

**Comment by stigrj** [comment]
The seed is given as input, so they should be the same

File: externalproject/src/my_random.h [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#pragma once
```
Comment (robertodr) [inline externalproject/src/my_random.h]: Can you try using C++11 PRNG? https://en.cppreference.com/w/cpp/numeric/random

**Comment by stigrj** [comment]
Nah, still different results on Mac

**Comment by robertodr** [comment]
Switiching to the Mersenne twister engine and a uniform real distribution seems to give same results on both platforms. Tests are failing because reference values are out of sync though. The default engine is implementation dependent and that might explain why the results are different on Linux and macOS.

**Comment by stigrj** [comment]
Thanks!

**Comment by robertodr** [comment]
No problem, at least now we know how to handle PRNGs properly with C++ :octopus:

File: Pipfile [hunk 3]
```diff
@@ -6,3 +6,4 @@ verify_ssl = true
 
 [packages]
 parselglossy = {git='https://github.com/dev-cafe/parselglossy.git'}
```
Comment (robertodr) [inline Pipfile]: We have a proper version on PyPI. Can you do `parselglossy = "*"` here and updated the lock file?

**Review by robertodr — APPROVED** [review]
Thanks for adding the tests!

**Pull request merged**